R/DESeqandFactoMineRforASOSpecificity.R

```diff
@@ -395,25 +396,10 @@ lockBinding("black.bold.18.text", globalenv())
   dim1 <-data.frame(res.desc$Dim.1)
   columns(org.Hs.eg.db)
   # TODO(dlroxe): Figure out why it is necessary to comment out
-  # the mapIDs calls, here and for Dim.2 below.  Perhaps they
-  # don't work (and are needless anyway) because the work was
-  # done when .addGeneIdentifiers() was called elsewhere?
-
-  #dim1$symbol = mapIds(org.Hs.eg.db,
-  #                     keys=row.names(dim1),
-  #                     column="SYMBOL",
-  #                     keytype="ENSEMBL",
-  #                     multiVals="first")
-  #dim1$entrez = mapIds(org.Hs.eg.db,
-  #                     keys=row.names(dim1),
-  #                     column="ENTREZID",
-  #                     keytype="ENSEMBL",
-  #                     multiVals="first")
-  #dim1$name =   mapIds(org.Hs.eg.db,
-  #                     keys=row.names(dim1),
-  #                     column="GENENAME",
-  #                     keytype="ENSEMBL",
-  #                     multiVals="first")
+  # gene identifier assignment, here and for Dim.2 below.
```

Comment:
I am not sure why this is showing up as a diff here.  It seems the same as a change that was already merged in pull 29.
https://github.com/a3609640/DNFA-gene-analysis/pull/29/files